package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "phantom-cabal",
+  "version": "1.0.0",
+  "description": "transport-phantom",
+  "main": "index.js",
+  "scripts": {
+    "start": "node server.js",
+    "dev": "nodemon --exec babel-node server.js"
+  },
+  "author": "Niyiragira",
+  "license": "ISC",
+  "dependencies": {
+    "body-parser": "^1.19.0",
+    "express": "^4.17.1",
+    "swagger-jsdoc": "^4.0.0",
+    "swagger-ui-express": "^4.1.4"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.11.6",
+    "@babel/core": "^7.11.6",
+    "@babel/node": "^7.10.5",
+    "@babel/preset-env": "^7.11.5",
+    "dotenv": "^8.2.0",
+    "eslint": "^7.9.0",
+    "eslint-plugin-react": "^7.21.0",
```

Comment:
What's the use of this package?

Reply:
when I was configuring ESLINT they asked me if I want to install that. And as I knew we were going to use react in future ,I installed it. But I have not yet used that up to now

Reply:
You don't need it now, remove it.